pkg/codegen/testing/test/testdata/regress-py-14012/python/pulumi_foo/_inputs.py

```diff
@@ -0,0 +1,53 @@
+# coding=utf-8
+# *** WARNING: this file was generated by test. ***
+# *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+import copy
+import warnings
+import pulumi
+import pulumi.runtime
+from typing import Any, Callable, Mapping, Optional, Sequence, Union, overload
+from . import _utilities
+
+__all__ = [
+    'ProviderCertmanagerArgs',
+]
+
+@pulumi.input_type
+class ProviderCertmanagerArgs:
+    def __init__(__self__, *,
+                 mtls_cert_pem: pulumi.Input[str],
+                 mtls_key_pem: pulumi.Input[str]):
+        ProviderCertmanagerArgs._configure(
+            lambda key, value: pulumi.set(__self__, key, value),
+            mtls_cert_pem=mtls_cert_pem,
+            mtls_key_pem=mtls_key_pem,
+        )
+    @staticmethod
+    def _configure(
+             _setter: Callable[[Any, Any], None],
+             mtls_cert_pem: pulumi.Input[str],
+             mtls_key_pem: pulumi.Input[str],
+             opts: Optional[pulumi.ResourceOptions]=None):
```

Comment:
@dixler, I kind of get why `ProviderArgs._configure` has an `opts` parameter, but I don't understand why any other type like this needs it. Seems like it's not necessary here.

Reply:
if it's being supplied via kwargs, we need to catch it iirc.

Reply:
> if it's being supplied via **kwargs, we need to catch it iirc.

I get it for top-level resource args.

But this is a nested object. I'm not seeing how opts would get passed to that.